audio-api/src/main/scala/no/ndla/audioapi/AudioApiProperties.scala

```diff
@@ -36,6 +37,13 @@ class AudioApiProperties extends BaseProps with DatabaseProps with StrictLogging
   val StorageName: String           = propOrElse("AUDIO_FILE_S3_BUCKET", s"$Environment.audio.ndla")
   val StorageRegion: Option[String] = propOrNone("AUDIO_FILE_S3_BUCKET_REGION")
 
+  val TranscribeStorageName: String = propOrElse("TRANSCRIBE_FILE_S3_BUCKET_NAME", s"$Environment.transcribe.ndla")
```

Comment:
Kanskje bruke samme format som for filer.
```suggestion
  val TranscribeStorageName: String = propOrElse("TRANSCRIBE_FILE_S3_BUCKET", s"$Environment.transcribe.ndla")
```